explore/src/main/scala/explore/SideTabs.scala

```diff
@@ -25,6 +25,21 @@ object SideTabs {
 
   implicit val propsReuse: Reusability[Props] = Reusability.derive
 
+  // This is a protected method on cats.Foldable. If we add
+  // fs2 to the project, we can switch to intersperse on fs2.Stream
+  def intersperseList[A](xs: List[A], x: A): List[A] = {
```

Comment:
I **think** that after we have a `List[List[A]]`, we can use cat's `intercalate` with a `List(x)` to achieve this.

Also, `fs2` is available since it's imported transitively. But we could add it explicitly if we are going to build streams. IDK what the right criteria would be for explicitly declaring libs

Reply:
At the point where we call intersperse, we have a `List[TagMod]`, not a `List[List[TagMod]]`. I could make it work by having `makeButtonSection` return `List[TagMod]` instead of a `TagMod`, bit that feels wrong to me.

I didn't think about fs2 being pulled in transitively.

Reply:
`mkTagMod` was right there in the VDOM documentation. Thanks for thinking of it, Carlos. That cleaned things up nicely.